templates/password/layout.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html language="de">
+
+<head>
+  {% block head %}
+  {% endblock %}
+</head>
+
+<body>
+  {% block body %}
+  {% endblock %}
+</body>
+
+</html>
```

Comment:
There are some files without newlines at the end in this PR. Please consider changing this.